samples/ImageBadgeSample.js

```diff
@@ -5,25 +5,37 @@ enyo.kind({
 	components: [
 		{kind: "moon.Scroller", fit: true, components: [
 			{kind: "moon.Divider", content: "Image Badges:"},
-			{kind: "moon.Image", src: "http://placehold.it/340x360&text=Image+One", alt: "Image One", components: [
+			{kind: "moon.Image", src: {
+				"hd" : "http://placehold.it/228x240&text=Image+One",
+				"fhd": "http://placehold.it/342x360&text=Image+One"
+			}, alt: "Image One", components: [
 				{kind: "moon.Icon", icon: "check"},
 				{kind: "moon.Icon", icon: "closex"},
 				{kind: "moon.Icon", icon: "drawer", classes: "float-right"}
 			]},
-			{kind: "moon.Image", src: "http://placehold.it/180x240&text=Image+Two", alt: "Image Two", components: [
+			{kind: "moon.Image", src: {
+				"hd" : "http://placehold.it/120x160&text=Image+Two",
+				"fhd": "http://placehold.it/180x240&text=Image+Two"
+			}, alt: "Image Two", components: [
 				{kind: "moon.Icon", icon: "closex"}
 			]},
 			
 			{kind: "moon.Divider", classes: "image-badge-sample-divider", content: "Image Badges - Show on Spotlight:"},
 			{kind: "moon.Item", components: [
-				{kind: "moon.Image", src: "http://placehold.it/340x360&text=Image+One", alt: "Image One", showBadgesOnSpotlight: true, components: [
+				{kind: "moon.Image", src: {
+					"hd" : "http://placehold.it/228x240&text=Image+One",
+					"fhd": "http://placehold.it/342x360&text=Image+One"
+				}, alt: "Image One", showBadgesOnSpotlight: true, components: [
 					{kind: "moon.Icon", icon: "check"},
 					{kind: "moon.Icon", icon: "closex"},
 					{kind: "moon.Icon", icon: "drawer", classes: "float-right"}
 				]}
 			]},
 			{kind: "moon.Item", components: [
-				{kind: "moon.Image", src: "http://placehold.it/180x240&text=Image+Two", alt: "Image Two", showBadgesOnSpotlight: true, components: [
+				{kind: "moon.Image", src: {
```

Comment:
Agreed - I guess I'm kind of torn as we had made changes to other samples to utilize a hash for the `src` property and I'm wondering if we should also re-add demonstrations of the single string `src` to those samples. We had made changes in these samples (including this PR) specifically to address issues with image scaling at 720p, as there are other samples with instances of images where we're using the single string `src`. I think that was kind of the idea of having all of the variations demonstrated in the `ImageSample`, as that sample is specifically concerning `moon.Image`, rather than trying to exhaustively demonstrate various aspects of secondary controls in a given sample. I don't think I'm necessarily opposed to re-adding a single `src` demonstration to just this sample, though, as it is closely related to `moon.Image`.